src/Microsoft.TestPlatform.CommunicationUtilities/DataCollectionRequestHandler.cs

```diff
@@ -282,6 +282,19 @@ public void ProcessRequests()
                         }
 
                         break;
+
+                    case MessageType.TestHostInitialized:
+                        if (EqtTrace.IsInfoEnabled)
```

Comment:
Add test for this.

Reply:
updated DataCollectionRequestHandlerTests.cs

Reply:
Done